frontend/src/pages/integrations/railway/create.tsx

```diff
@@ -20,15 +21,27 @@ import {
 import { useGetWorkspaceById } from "../../../hooks/api/workspace";
 import createIntegration from "../../api/integrations/createIntegration";
 
+interface RailwayCreateIntegrationFormValues {
```

Comment:
Instead of typing the types, we can use yup to set the validation types and generate the types from there.

Reply:
REF: https://github.com/Infisical/infisical/blob/main/frontend/src/views/Settings/OrgSettingsPage/components/OrgNameChangeSection/OrgNameChangeSection.tsx#L11